app/services/fake-data.service.ts

```diff
@@ -47,157 +47,156 @@ export function generateSchedule(): Array<IMainEventSessions> {
+			accomodation: EventList[i].accomodation,
+			transportation: EventList[i].transportation,
+			contantInfo: EventList[i].contantInfo 
+		}
+		eventList.push(s);
+	}
+	return eventList;
 }
 
 export function generateSessions(speakers: Array<ISpeaker>, roomInfos: Array<IRoomInfo>): Array<ISession> {
-    let sessionList: Array<ISession> = [];
-    let idSeed = 1000;
-    for (let confDay of sessionDays) {
-        let timeSlots = generateTimeSlots(confDay);
-        for (let confTimeSlot of timeSlots) {
-            if (confTimeSlot.isBreak) {
-                let s: ISession = {
-                    id: (idSeed++).toString(),
-                    title: toTitleCase(confTimeSlot.title),
-                    isBreak: true,
-                    start: confTimeSlot.start.toString(),
-                    end: confTimeSlot.end.toString(),
-                    room: "",
-                    roomInfo: null,
-                    speakers: [],
-                    description: "",
-                    descriptionShort: "",
-                    type: "",
-                    floor: "",
-                };
-                sessionList.push(s);
-            }
-            else {
-                let subSpeakers = getRandomArrayElements(speakers, faker.random.number({ min: 1, max: 3 }));
-                let roomInfo = roomInfos[faker.random.number(roomInfos.length - 1)];
-                var randomTypeIndex = Math.floor(Math.random() * SessionTypes.length); 
-                var randomFloorIndex = Math.floor(Math.random() * SessionFloor.length);
-                let s: ISession = {
-                    id: (idSeed++).toString(),
-                    title: toTitleCase(faker.company.bs()),
-                    isBreak: false,
-                    start: confTimeSlot.start.toString(),
-                    end: confTimeSlot.end.toString(),
-                    room: roomInfo.name,
-                    roomInfo: roomInfo,
-                    speakers: subSpeakers,
-                    description: faker.lorem.paragraph(8),
-                    descriptionShort: faker.lorem.sentence(),
-                    type: SessionTypes[randomTypeIndex],
-                    floor: SessionFloor[randomFloorIndex],
-                };
-                sessionList.push(s);
-            }
-        }
-    }
-    return sessionList;
+	let sessionList: Array<ISession> = [];
+	let idSeed = 1000;
+	for (let confDay of sessionDays) {
+		let timeSlots = generateTimeSlots(confDay);
+		for (let confTimeSlot of timeSlots) {
+			if (confTimeSlot.isBreak) {
+				let s: ISession = {
+					id: (idSeed++).toString(),
+					title: toTitleCase(confTimeSlot.title),
+					isBreak: true,
+					start: confTimeSlot.start.toString(),
+					end: confTimeSlot.end.toString(),
+					room: "",
+					roomInfo: null,
+					speakers: [],
+					description: "",
+					descriptionShort: "",
+					type: "",
+					floor: "",
```

Comment:
Tried to readjust the indentation, it looks fine on my VS Code (using tab size 2), not sure why its so obscure here...